### DISCUSSION TITLE
Is anyone using "Cusdis" comment system in this theme? I have a blank area

### DISCUSSION THREAD

**Discussion by miazme** [nunocoracao/blowfish#28] [Q&A] [answered]
I used to have [Cusdis](https://cusdis.com/) as my comment system in another hugo theme, so there is nothing wrong with the Cusdis configuration. 

But when I try to embed Cusdis code in blowfish(as the doc, create `partials/comments.html` and set `showComments = true`), there is no input box whatsoever. I have no clue what's wrong and what can I do.

---
It supposed to be look like this:
<img width="727" alt="comments-look" src="https://user-images.githubusercontent.com/95271776/192708355-96c2247e-b03a-48de-ad2c-591416b057ba.png">

but when I embed the same code to `partials/comments.html`
<img width="1133" alt="comments-code" src="https://user-images.githubusercontent.com/95271776/192708563-386a686a-4d60-4251-9fe7-cfa62760a1cc.png">

I got a blank area:
<img width="712" alt="comments-error" src="https://user-images.githubusercontent.com/95271776/192708628-90efb9a1-479c-4669-8349-a0bef218a506.png">

looking forward to the solution. 🙏

**Comment by nunocoracao** [comment]
@miawithcode sorry haven't tried it. I can have a look when you have time if you have the code available in a repo. share the URL if you want to

**Reply by miazme** [reply]
@nunocoracao hi, I create a repo and here's the code: https://github.com/miawithcode/cusdis/blob/main/comments.html

**Comment by nunocoracao** [comment, marked as answer]
Found a solution @miawithcode 

Changed your file a little bit to import the js from cusdis in a "safer" way. The problem was a clash between that code and the theme core js files. Opened a PR to the file you shared so that you can see the changes in detail.

```html
<h3>Comment</h3>
<div id="cusdis_thread"
  data-host="https://cusdis.com"
  data-app-id="34a5cbd9-4203-4b8f-b555-742871da8c60"
  data-page-id="{{ .File.UniqueID }}"
  data-page-url="{{ .RelPermalink }}"
  data-page-title="{{ .Title }}"
></div>
{{- $js := resources.GetRemote "https://cusdis.com/js/cusdis.es.js" | js.Build (dict "minify" true) -}}
<script type='text/javascript'>
  {{ $js.Content | safeJS }}
</script>
```

![Screen Shot 2022-10-02 at 00 42 30](https://user-images.githubusercontent.com/3803196/193432080-cdad8e11-13ef-4ed0-8521-d6feb8a347ad.png)

**Reply by miazme** [reply]
@nunocoracao That's great. Thanks so much!
